index.js

```diff
@@ -6,10 +6,13 @@ const HapiAlive = require('hapi-alive')
 const HapiDevErrors = require('hapi-dev-errors')
 const server = new Hapi.Server()
 
+// Load application configuration using Dotenv (see https://www.npmjs.com/package/dotenv)
+require('dotenv').config()
+
 const environment = process.env.NODE_ENV
```

Comment:
Not a change to this line, but can be include an entry for this in the `.env.example` file.